Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -135,7 +171,7 @@ - (BOOL)useSimulatorWithDeviceUDID:(NSUUID *)deviceUDID {
 
 - (void)bootWithCompletion:(void (^)(NSError *error))completion {
     // Now boot it.
-    [BPUtils printInfo:INFO withString:@"Booting a simulator without launching Simulator app"];
+    [BPUtils printInfo:INFO withString:@"Done with create simulator and start booting without launching Simulator app"];
```

Comment:
Any reason for the change in the message?

Reply:
This is to indicate when create simulator is done, so that you can read from log how long create simulator takes